cmd/wallet-sdk-js/jsinterop/agent.go

```diff
@@ -101,21 +104,36 @@ func CreateDID(_ js.Value, args []js.Value) (any, error) {
 		return nil, err
 	}
 
-	content, err := didDoc.JSONBytes()
+	return types.SerializeDIDDoc(didDoc)
+}
+
+func CreateOpenID4CIInteraction(_ js.Value, args []js.Value) (any, error) {
+	if agentInstance == nil {
+		return nil, walleterror.NewExecutionError(
+			errors.Module,
+			errors.InitializationFailedCode,
+			errors.InitializationFailedError,
+			fmt.Errorf("agent instance is not initialized"))
+	}
+
+	initiateIssuanceURI, err := jssupport.EnsureString(jssupport.GetNamedArgument(args, "initiateIssuanceURI"))
+	if err != nil {
+		return nil, err
+	}
+
+	interaction, err := agentInstance.CreateOpenID4CIInteraction(initiateIssuanceURI)
```

Comment:
If my PR gets merged in first, can you rename this (and anything else referencing interactions) from `Interaction` to `IssuerInitiatedInteraction`? (If your PR gets merged in first, I will do the renaming)

Reply:
yes, sure

Reply:
Also, we'll need to add another one for the `WalletInitiatedInteraction`

Reply:
I think we will have a new issue for this.

Reply:
WalletInitiatedInteraction is out of the scope of this PR. Currently, we working on PreAuth only